### PR TITLE
[pkp-lib][main] #5697 Disable GalleyForms and DependentFiles Grid

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: controllers/wizard/fileUpload/FileUploadWizardHandler.php [hunk 1]
```diff
@@ -438,8 +440,13 @@ public function uploadFile($args, $request)
...
     public function editMetadata($args, $request)
     {
+        $stageId = $this->getStageId();
+        if (!$stageId) {
+            $stageId = (int) $request->getUserVar('stageId');
+        }
+
         $submissionFile = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION_FILE);
-        $form = new SubmissionFilesMetadataForm($submissionFile, $this->getStageId(), $this->getReviewRound());
+        $form = new SubmissionFilesMetadataForm($submissionFile, $stageId, $this->getReviewRound());
```
Comment (NateWr) [inline controllers/wizard/fileUpload/FileUploadWizardHandler.php:449]: Hmm, we seem to be passing in a stage ID directly from the request, without validating it here. These changes don't appear in the stable branch fixes. Can you explain more about why the changes to `FileUploadWizardHandler` are required?

Comment (defstat) [reply controllers/wizard/fileUpload/FileUploadWizardHandler.php:449]: if you see [here](https://github.com/pkp/pkp-lib/pull/8228/files/4a8f8ebcd8395bf35c845a4521a775646aee8c4c#diff-4852f38c94c8e20e1e29c33ec68b0f9db140ec183104b57ccec3ce95f6f05ba3R443-R446) I am trying to use either the getStageId if the authorized context exists, and if it does not, to get the stageId from the request.

That was a way to pass a problem I encountered regarding `DependentFilesGrid` (see bellow)
image

Without this change, the authorized context object with assoc type `ASSOC_TYPE_WORKFLOW_STAGE` does not exist, but the `stageId` parameter does. 

This lines of code should be removed, I think, and a bug issue should be added (if there is no relevant issue) to deal with that independently.

Comment (NateWr) [reply controllers/wizard/fileUpload/FileUploadWizardHandler.php]: @asmecher can you make a call on how to handle this? Other than this, I'm happy for the rest of the PRs to be merged (stable and main).

Comment (asmecher) [reply controllers/wizard/fileUpload/FileUploadWizardHandler.php]: It looks like this already got removed -- was the problem resolved another way or will it still need attention?

Comment (asmecher) [reply controllers/wizard/fileUpload/FileUploadWizardHandler.php]: Checked in with @defstat -- he removed that part of the code and will file the issue separately. So this is ready to go; I'll merge.

Comment (defstat) [reply controllers/wizard/fileUpload/FileUploadWizardHandler.php]: Filled #8347 issue
